runtime/ops/tty.rs

```diff
@@ -126,9 +126,14 @@ fn op_stdin_set_raw(
         use libc::tcsetattr;
         use libc::termios;
 
+        static mut ORIG_TERMIOS: Option<termios> = None;
```

Comment:
Can this method be called from a worker? If so, I think that makes this unsafe.

```
const src = new Blob([`Deno.stdin.setRaw()`]);
new Worker(URL.createObjectURL(src), { type: "module" });
```

I think it would just be better to throw it all in a OnceCell.